groups/templates/groups/discussion_thread_base.html

```diff
@@ -20,8 +20,15 @@
                 <ul id="{{ comment.get_pagejump_anchor }}">
                     <li>
                         {{ comment.user }} wrote at <a href="{{ comment.get_pagejump }}">{{ comment.date_created }}</a>:
+                        {% if comment.user_may_delete %}(<a href="{% url 'comment-delete' pk=comment.pk %}">delete this comment</a>){% endif %}
+                    </li>
+                    <li>
+                        {% if comment.is_deleted %}
```

Comment:
That's true.

The only real difference is in the data presented when this line is forgotten in an update. In my example, it would erroneously say "Post deleted", but in yours, it would erroneously display the post.

It's something of a contrived example, tbh. I can imagine states that displaying rather than hiding the data is best.